docs/wallet_API.md

```diff
@@ -80,7 +80,11 @@ The Beacon wallet requires an extra step to set up the network to connect to and
 await wallet.requestPermissions();
 ```
 
-In previous versions of Beacon, you were able to set the `network` property when doing `requestPermissions()`. This behaviour was removed from Beacon, and you must now set the network when instantiating the wallet.
+:::note Subscribe to Events to be notified of changes in the wallet
```

Comment:
From beacon doc I found them `dAppClient.subscribeToEvent()` then requestPermission(). Might be helpful for user to see this paragraph about subscribe before line 80 of `await wallet.requestPermissions();`

Reply:
the goal of subscribe to event change is to transition all users to start using them, if we don't give them the right order will defeat the purpose

Reply:
I don't see any problems with including them in this part, it probably is better for consistency. Having one part mention the events, and the other not will send mixed messages to readers

Reply:
but I also understand the separation since there's a section specifically for events

Reply:
Added a comment that reminds the user that they should subscribe to events.